dfx.json

```diff
@@ -3,10 +3,8 @@
     "icp_hello_world_rust_backend": {
       "candid": "src/icp_hello_world_rust_backend/icp_hello_world_rust_backend.did",
       "package": "icp_hello_world_rust_backend",
-      "type": "custom",
-      "shrink": true,
-      "wasm": "icp_hello_world_rust_backend.wasm",
-      "build": "scripts/build.sh"
+      "type": "rust",
+      "post_install": ["cp target/wasm32-unknown-unknown/release/icp_hello_world_rust_backend.wasm icp_hello_world_rust_backend.wasm"]
```

Comment:
Why do you need this?

Reply:
Good point. PocketIC was set up such that it expects it there, but there's no need for that. I've updated it!